src/components/shared/Entity/Details/Overview.tsx

```diff
@@ -0,0 +1,38 @@
+import { Grid } from '@mui/material';
+import { DataPreview } from 'components/collection/DataPreview';
+import { useEditorStore_currentCatalog } from 'components/editor/Store/hooks';
+import { useEntityType } from 'context/EntityContext';
+import useGlobalSearchParams, {
+    GlobalSearchParams,
+} from 'hooks/searchParams/useGlobalSearchParams';
+import ShardInformation from '../Shard/Information';
+
+function Overview() {
+    const entityType = useEntityType();
+    const catalogName = useGlobalSearchParams(GlobalSearchParams.CATALOG_NAME);
+    const isCollection = entityType === 'collection';
+
+    const currentCatalog = useEditorStore_currentCatalog({
+        localScope: true,
+    });
+    const catalogSpec = currentCatalog?.spec ?? null;
+    const isDerivation = Boolean(catalogSpec?.derivation);
```

Comment:
@psFried - wanted a quick sanity check. To hide the shard status for derivations instead of trying to fetch shard status I was thinking we could just check if the specification contains a `derivation`.